app/build.gradle

```diff
@@ -42,12 +41,12 @@ dependencies {
     implementation fileTree(dir: "libs", include: ["*.jar"])
     implementation project(':fearless-utils')
     implementation "com.google.android.material:material:1.8.0"
-    implementation 'androidx.core:core-ktx:1.9.0'
-    implementation 'androidx.activity:activity-compose:1.6.1'
+    implementation 'androidx.core:core-ktx:1.10.0'
+    implementation 'androidx.activity:activity-compose:1.7.1'
 
-    implementation "androidx.compose.ui:ui:1.3.3"
-    implementation "androidx.compose.material:material:1.3.1"
-    implementation "androidx.compose.ui:ui-tooling-preview:1.3.3"
-    androidTestImplementation "androidx.compose.ui:ui-test-junit4:1.3.3"
-    debugImplementation "androidx.compose.ui:ui-tooling:1.3.3"
+    implementation "androidx.compose.ui:ui:1.4.2"
```

Comment:
Google recommends to use [compose-bom dependency](https://developer.android.com/jetpack/compose/bom) because it will make it easier to use all of the latest stable Compose versions at the same time. Possibly you could to add this dependency in this PR and remove custom versions for all compose dependencies

Reply:
it's sample app. no need to worry